docs/CONFIG-VARS.md

```diff
@@ -419,7 +419,7 @@ The EBS CSI driver is currently only used for kubernetes v1.23 or later AWS EKS
 | INGRESS_NGINX_NAMESPACE | NGINX Ingress Helm installation namespace | string | ingress-nginx | false | | baseline |
 | INGRESS_NGINX_CHART_URL | NGINX Ingress Helm chart URL | string | See [this document](https://kubernetes.github.io/ingress-nginx) for more information. | false | | baseline |
 | INGRESS_NGINX_CHART_NAME | NGINX Ingress Helm chart name | string | ingress-nginx | false | | baseline |
-| INGRESS_NGINX_CHART_VERSION | NGINX Ingress Helm chart version | string | "" | false | If left as "" (empty string), version `4.11.1` is used for Kubernetes clusters whose version is >= 1.26.X, and for Kubernetes clusters whose version is <= 1.25.X please set this variable to avoid errors. See [Supported Versions table](https://github.com/kubernetes/ingress-nginx/?tab=readme-ov-file#supported-versions-table) for the supported versions list. | baseline |
+| INGRESS_NGINX_CHART_VERSION | NGINX Ingress Helm chart version | string | "" | false | If left as "" (empty string), version `4.12.0` is used for Kubernetes clusters whose version is >= 1.26.X, for Kubernetes clusters whose version is <= 1.25.X you must set this variable to avoid errors. See [Supported Versions table](https://github.com/kubernetes/ingress-nginx/?tab=readme-ov-file#supported-versions-table) for the supported versions list. | baseline |
```

Comment:
I believe the k8s cluster version warning needs to be adjusted. ingress-nginx 1.12.0 is documented to support k8s 1.28 to 1.32.

Reply:
Thanks @ephesused, I also needed to update the k8sMinorVersion value for the new ingress-nginx version so that v1.12.0 will only be installed by default for k8s 1.28 and newer clusters. The output message and behavior should indicate that for K8s clusters <= v1.27, the INGRESS_NGINX_CHART_VERSION var must be explicitly set.

Reply:
@ephesused , ready for re-review when you get a chance.